template/build/webpack.prod.conf.js

```diff
@@ -63,7 +59,7 @@ const webpackConfig = merge(baseWebpackConfig, {
     // you can customize output by editing /index.html
     // see https://github.com/ampedandwired/html-webpack-plugin
     new HtmlWebpackPlugin({
-      filename: {{#if_or unit e2e}}process.env.NODE_ENV === 'testing'
+      filename: {{#if_or unit e2e}}process.env.VUE_TEST = 'e2e'
```

Comment:
See `/test/e2e/runner.js`: 

There we set `NODE_ENV` to `test`, but we use the wbepack dev server in the `webpack.prod.conf.js`, because the want to run e2e tests against a devserver that serves likes like in production. Therefore, we have to use a second env variable as a workaround to make the production config behave like `NODE_ENV === 'production'`